From 46f078dba7fbfa4f1effde2ebec5c92794fe3be6 Mon Sep 17 00:00:00 2001
Message-ID: <46f078dba7fbfa4f1effde2ebec5c92794fe3be6.1785954953.git.mjg@fedoraproject.org>
From: Michael J Gruber <mjg@fedoraproject.org>
Date: Tue, 30 Jun 2026 12:59:48 +0200
Subject: [PATCH] pdf-object.c: Include UINT_MAX definition and fix FTBFS

59aed0383 ("Bug 709343: Speed up mutool clean garbage collection by hashing objects.", 2026-06-24)
introduced usage of UINT_MAX in this file, so include the definition.
---
 source/pdf/pdf-object.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/source/pdf/pdf-object.c b/source/pdf/pdf-object.c
index c7ecd101b..08e1a924a 100644
--- a/source/pdf/pdf-object.c
+++ b/source/pdf/pdf-object.c
@@ -27,6 +27,7 @@
 #include <stdarg.h>
 #include <stdlib.h>
 #include <string.h>
+#include <limits.h>
 #include <math.h>
 
 #define PDF_MAKE_NAME(STRING,NAME) STRING,
-- 
2.55.0.607.ge54b5b6513

